debian/rules

```diff
@@ -18,4 +18,7 @@ export DH_OPTIONS
 
 
 %:
-	dh $@ 
+	dh $@
+
+override_dh_auto_configure:
+	./autogen.sh
```

Comment:
If you make this line
        ./autogen.sh --prefix=/usr
then you can probably leave the .install files unchanged.